tomviz/python/tomviz/logger.py

```diff
@@ -0,0 +1,146 @@
+from tomviz.io import dm

```

Comment:
As this file is specfic to the operator is should just live next to it in `tomviz/python`. The tomviz python package is for modules that could be used across tomviz.